packages/openapi-to-graphql/README.md

```diff
@@ -137,34 +137,52 @@ Notice that the slashes in the path `/favoriteBooks/{name}` must be escaped with
 The `createGraphQlSchema` function takes an optional `options` object as a second argument:
 
 ```javascript
-createGraphQLSchema(oas, [options])
```

Comment:
I thought the brackets were kind of confusing so I removed them. What do you think?

Reply:
Looks good.